ui/src/components/chat/sidebar.rs

```diff
@@ -314,6 +343,11 @@ pub fn Sidebar(cx: Scope<Props>) -> Element {
                     let other_participants =  state.read().remove_self(&participants);
                     let user: state::Identity = other_participants.first().cloned().unwrap_or_default();
                     let platform = user.platform().into();
+                    let is_group_conv = participants.len() > 2;
```

Comment:
I added the field `conversation_type` to `State::Chat`. We should use that (and compare it to `ConversationType::Group`) instead of checking the number of participants. Especially if in the future there's the option to have group chats with 1 participant.